prebid-server/features/pbs-module-storage.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page_v2
+sidebarType: 5
+title: Prebid Server | Features | Module Storage
+---
+
+# Prebid Server | Features | Module Storage
```

Comment:
Questions:

- isn't this document only relevant to module authors?
- would this info better be placed in https://docs.prebid.org/prebid-server/developers/add-a-module-java.html ? It's Java specific, and seems like having all of this in one place would be good?

Reply:
Discussed adding a small section for what PBS Host Companies need to do. 

Put all the other sections here under a "developer" section, and point to this doc from the module developers doc.

Reply:
Discussed renaming this "Prebid Cache Storage" to perhaps generalize beyond modules someday.